deploy/linux/deploy.bash

```diff
@@ -24,6 +24,8 @@ fi
 AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:-us-west-2}
 AWS_DEFAULT_OUTPUT=${AWS_DEFAULT_OUTPUT:-text}
```

Comment:
Should these be exported too?

Reply:
I have no idea. I assume the values aren't really being used -- there's no other reference to them anywhere in our scripts. But maybe the aws cli looks for them? All I know at the moment is that the change is working and I'm not inclined to keep messing with it... ;-)

Reply:
Those variables are standard awscli variables. If those variables were previously exported, we might not have had the problem in the first place. But, yes, I'm also fine with leaving this as is since it is working.